### PR TITLE
Partial like 

### PR DESCRIPTION
**Pull request by Rhaypapenfuzz**

### CODE REVIEW

File: src/card.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ import CardContent from "@material-ui/core/CardContent";
 import CardMedia from "@material-ui/core/CardMedia";
 import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
-
-import Paper from "@material-ui/core/Paper";
+import Firestore from './services/firestore.js';
+import firebaseConfig from './services/firestore.js'; // Must be present to initialize database.
```
Comment (kricheso) [inline src/card.js:11]: Please remove this import line. Only needed in script/test files.

File: src/card.js [hunk 2]
```diff
@@ -19,8 +19,45 @@ const useStyles = makeStyles({
   },
 });
 
-function SingleCard({ name, author, gameLink, numberOfLikes }) {
+
+function SingleCard({ name, author, gameLink, numberOfLikes, creatorId, gridId }) {
```
Comment (kricheso) [inline src/card.js]: Optional but advised: use conventional method where the parameter is called props.

Comment (Rhaypapenfuzz) [reply src/card.js]: passing properties this way is pretty popular too

**Review by kricheso — APPROVED** [review]
Approved.

**Pull request merged**